### PR TITLE
DFR-2461: send order event

### PR DESCRIPTION
**Pull request by srochani**
### Checklist

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)
- [ ] Does this PR introduce a breaking change

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/helper/DocumentHelper.java [hunk 1]
```diff
@@ -607,6 +607,14 @@ public <T> T deepCopy(T object, Class<T> objectClass) {
         }
     }
 
+    public <T> T deepCopyArray(T object, TypeReference<T> typeReference) {
+        try {
```
Comment (rishikrsharma) [inline src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/helper/DocumentHelper.java:611]: We have DeepCopy method already in this class. Use that.

File: src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/sendorder/SendOrderApplicantDocumentHandler.java [hunk 2]
```diff
@@ -96,9 +100,11 @@ private static void setConfidentialBulkPrintFieldForApplicant(FinremCaseDetails
     }
 
     protected void setConsolidateCollection(FinremCaseData caseData, List<ApprovedOrderCollection> orderCollection) {
+        List<ApprovedOrderCollection> orderCollectionCopy = documentHelper.deepCopyArray(orderCollection,
+            new TypeReference<List<ApprovedOrderCollection>>() {});
```
Comment (rishikrsharma) [inline src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/sendorder/SendOrderApplicantDocumentHandler.java:104]: why we need to make this copy?

File: src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/helper/DocumentHelper.java (hunk 1 above)
Comment (srochani) [reply src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/helper/DocumentHelper.java:611]: DeepCopy method is for object not list of array so I have created new one for array

File: src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/sendorder/SendOrderApplicantDocumentHandler.java (hunk 2 above)
Comment (srochani) [reply src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/sendorder/SendOrderApplicantDocumentHandler.java:104]: Here, we have multiple collection, If we don't copy then same object reference is use and if we apply category to one collection then it get refer in other collection

**Review by rishikrsharma — APPROVED** [review]

**Pull request merged**